src/components/Observations/MapLegend.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box, Typography, useTheme } from '@mui/material';
+import { useDeviceInfo } from '@terraware/web-components/utils';
+
+export type MapLegendItem = {
+  borderColor: string;
+  fillColor: string;
+  label: string;
+};
+
+export type MapLegendProps = {
+  title: string;
+  items: MapLegendItem[];
+};
```

Comment:
The maps dashboard has multiple legends. Since this component also accounts for the background etc.. thoughts on making the title/items pair and array of title/items?

Reply:
Oh, I see what you mean. Yes, that would make sense to do I think. Thanks!